lib/src/st2110/pipeline/st20_pipeline_rx.c

```diff
@@ -471,6 +472,7 @@ static int rx_st20p_create_transport(struct mtl_main_impl* impl, struct st20p_rx
   ops_rx.rx_burst_size = ops->rx_burst_size;
   ops_rx.notify_frame_ready = rx_st20p_frame_ready;
   ops_rx.notify_event = rx_st20p_notify_event;
+  ops_rx.notify_detected = ops->notify_detected;
```

Comment:
It should define a `rx_st20p_notify_detected` function here to pass the correct `priv` data to the app as `ops_rx.priv = ctx;`